### ISSUE TITLE
Sticky corners

### ISSUE THREAD

**Issue by Martinligabue** [directvt/vtm#632]
I think it would be a nice feature to have the terminals stick to the corners when dragged to those, maybe even the sides?

**Comment by o-sdn-o** [comment]
I agree, it would be a useful feature. I'm going to put it on the backlog. Thanks for submitting this!
